content/docs/user-guide/assets/pipeline/metadata.md

```diff
@@ -0,0 +1,93 @@
+
+It is highly recommended that this setreg file be placed in a central location, which is shared with other users of the project, to ensure everyone is generating the same metadata files.  For a game project, it should be placed in the project directory; for gem development, it should be placed in the gem directory.  Keep in mind this is an Asset Processor (AP) wide setting, it will apply to all files the AP processes, not just the ones in the project or gem where the setting exists.
+
+Once a metadata file is generated for an asset and the asset is relocated or any references are saved using the newly generated UUID, the metadata file must be kept alongside the asset.  Not doing so could break references to the file.
+
+> AssetProcessor will continue to use existing metadata files for assets which have them, regardless of the above settings.
+
+## Renaming while AP is running
+By default, renaming assets manually while Asset Processor is running can prove difficult as AP will try to immediately create a new metadata file for the renamed asset.  To prevent this, AP can be configured to wait for a specified time before it attempts to process a "new" asset, which avoids the automatic creation of an unwanted, new metadata file.  
+
+> This delay will affect all newly created assets for metadata-enabled types and is off by default.  The longer the delay time, the longer it will take AP to start processing any newly created metadata-enabled file type.
+
+In a .setreg file, add the following setting:
+
+```json
+{
+    "O3DE":
+    {
+        "AssetProcessor":
+        {
+            "Settings":
+            {
+                "Metadata":
+                {
+                    "MetaCreationDelayMs": 5000
+                }
+            }
+        }
+    }
+}
+```
+The value specified is the milliseconds to wait.  The above example config will wait 5 seconds before processing an asset.  
+
+>  This setting can be user-specific and does not need to be shared across users if not desired.
+
+This setting is mainly useful to help allow manual renaming of assets, but can help in some situations of slow file updates on disk, such as copying a large number of files or fetching latest from source control.  These operations can sometimes be ordered in a way that requires a very high delay setting, such as a Git LFS fetch, which may fetch files in groups by type, possibly resulting in metadata files being created on disk long after the source asset was created.  In these situations, it is recommended to close AP first to avoid conflicts caused by AP creating metadata files which already exist in source control.
+
+## Asset Processor Batch
+Since Asset Processor Batch is intended for use with automated processes, it will not create metadata files and will not use the meta creation delay, regardless of what settings are configured.  It will make use of any existing metadata files however.
+
+## Limitations

```

Comment:
To learn more of the technical contribution/setup perhaps add a reference to the engine contributor doc here? Or cover in limitations ie. While feature is still experimental, see the engine contributor docs for further information?

Think it would be nice to highlight how others know more.

Reply:
Added a link to the contributors doc